elements/pl-dataframe/pl-dataframe.mustache

```diff
@@ -1,32 +1,43 @@
 {{#show_python}}
-<ul class="nav nav-tabs">
-  <li class="nav-item"><a class="nav-link active" data-toggle="tab" href="#table-{{uuid}}">Table</a></li>
-  <li class="nav-item"><a class="nav-link" data-toggle="tab" href="#code-{{uuid}}">Code</a></li>
-</ul>
+<script>
```

Comment:
Great to see this functionality incorporated from `pl-variable-output`.

Reply:
Have now moved this functionality to `pl-code`. With this change, I think that `pl-variable-output` should be reworked to use `pl-code` (since this will add syntax highlighting and simplify the code on their end).